src/data/guidelines/color-tokens.js

```diff
@@ -502,7 +502,7 @@ const colorTokens = {
       },
     },
     '$field-02': {
-      role: ['`Light` variant input fields', 'Fields on $ui-02 backgrounds'],
+      role: ['"Light" variant input fields', 'Fields on $ui-01 backgrounds'],
```

Comment:
Need to manually do smart quotes when it's not an `mdx` file.

```suggestion
      role: ['“Light” variant input fields', 'Fields on $ui-01 backgrounds'],
```